doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.isolatedStarAssociation.IsolatedStarAssociationTask.rst

```diff
@@ -0,0 +1,44 @@
+.. lsst-task-topic:: lsst.pipe.tasks.isolatedStarAssociation.IsolatedStarAssociationTask
+
+###########################
+IsolatedStarAssociationTask
+###########################
+
+``IsolatedStarAssociationTask`` matches unresolved stars from the ``sourceTable_visit`` datasets into a pure (but not complete) catalog of isolated stars.
+The output datasets are a catalog of isolated stars (dataset ``isolated_star_cat``) and the associated observations with configured columns from the input catalogs (dataset ``isolated_star_observations``).
```

Comment:
"and the associated observations" - "and a catalog of properties from each of the associated sources"? Trying to keep object/source terminology (since I think we use "source" where you're using "observation" here)? Which maybe suggests `isolated_star_observations` should be `isolated_star_sources`, or maybe I'm misunderstanding something.

I think "observation" in Rubin typically implies properties about the exposure itself, not the sources?

Reply:
I think you're right, and that's a good catch before things get embedded in the system.